### ISSUE TITLE
Authentication failed because "/" endpoint returns 403

### ISSUE THREAD

**Issue by cscetbon** [astefanutti/kubebox#21]
I can query my k8s cluster using my certiticates and key :
```
k cluster-info
Kubernetes master is running at https://......
Grafana is running at http://......
Heapster is running at https://.....
KubeDNS is running at https://..../proxy
Kubemonitor is running at http://.....
```
But kubebox complains and asks me for a username/password/token. 
<img width="1663" alt="screen_shot_2018-10-17_at_2_45_55_pm" src="https://user-images.githubusercontent.com/2619840/47109209-a6409380-d21b-11e8-8a3e-c0747d7fb18a.png">

Any workaround ?

**Comment by astefanutti** [comment]
May be related to #7.

**Comment by cscetbon** [comment]
@astefanutti yeah probably

**Comment by astefanutti** [comment]
Let me close this as it is followed-up in #7.

**Comment by astefanutti** [comment]
Kubebox assumes user is granted permission to discover master API endpoints, though that is not always the case, e.g.:

```
$ kubectl get --raw /
Error from server (Forbidden): forbidden: User "cscetbon" cannot get path "/"
```

**Comment by astefanutti** [comment]
@cscetbon could you check what returns the following:
```
$ kubectl get --raw /apis
```

**Comment by cscetbon** [comment]
@astefanutti https://pastebin.com/raw/qG5eM2aS

**Comment by astefanutti** [comment]
@cscetbon thanks. I just wanted to confirm you can access it. It should be possible to make the call to `/` optional in Kubebox.

**Comment by cscetbon** [comment]
Eager to test and see it in action 😉

**Comment by sabrehagen** [comment]
Me too. Do you have a fix in a branch we could test?

**Comment by astefanutti** [comment]
I plan to have this issue fixed for the upcoming release 0.3.2.

**Comment by astefanutti** [comment]
It should be fixed with 98b9c339058e45b8ed932ca356ece1b9210918ae. I'll cut a new release with the fix ASAP.
